components/glyphicons/glyphicons-en.html

```diff
@@ -0,0 +1,815 @@
+---
+title: Glyphicons
+description: Documentation and working examples for Glyphicons
+language: en
+altLangPage: glyphicons-fr.html
+dateModified: 2023-09-15
+layout: no-container
+---
+<div class="container">
+	<h1>{{ page.title }}</h1>
+
+	<div class="wb-prettify all-pre hide"></div>
+
+	<div class="row wb-eqht-grd">
+		<div class="col-lg-2 col-sm-3 col-xs-6 well">
+			<p>Asterisk<br><span class="glyphicon glyphicon-asterisk" aria-hidden="true"></span><br><code>glyphicon glyphicon-asterisk</code></p>
+		</div>
```

Comment:
Please either move the "well" class to the `<p>` tag or wrap the `<p>` tag with a `<div class="well">`. Do this for EN and FR.
```suggestion
		<div class="col-lg-2 col-sm-3 col-xs-6">
			<p class="well">Asterisk<br><span class="glyphicon glyphicon-asterisk" aria-hidden="true"></span><br><code>glyphicon glyphicon-asterisk</code></p>
		</div>
```